client/src/app/videos/+video-watch/video-watch.component.html

```diff
@@ -70,24 +70,24 @@ <h1 class="video-info-name">{{ video.name }}</h1>
                   </div>
 
                   <div
-                    [ngbPopover]="getRatePopoverText()" [ngClass]="{ 'activated': userRating === 'dislike' }" (click)="setDislike()"
-                    class="action-button action-button-dislike" role="button" [attr.aria-pressed]="userRating === 'dislike'"
+                    [ngbPopover]="getRatePopoverText()" [ngClass]="{ 'activated': userRating === 'dislike' }" (click)="setDislike()" (keyup.enter)="setDislike()"
+                    class="action-button action-button-dislike" tabindex="0" role="button" [attr.aria-pressed]="userRating === 'dislike'" [attr.aria-label]="tooltipDislike"
                     [ngbTooltip]="tooltipDislike"
                     placement="bottom auto"
                   >
                     <my-global-icon iconName="dislike"></my-global-icon>
                     <span *ngIf="video.dislikes" class="count">{{ video.dislikes }}</span>
                   </div>
 
-                  <div *ngIf="video.support" (click)="showSupportModal()" class="action-button action-button-support"
+                  <div *ngIf="video.support" (click)="showSupportModal()" (keyup.enter)="showSupportModal()" class="action-button action-button-support" tabindex="0" role="button" [attr.aria-label]="tooltipSupport"
                     [ngbTooltip]="tooltipSupport"
                     placement="bottom auto"
                   >
-                    <my-global-icon iconName="support"></my-global-icon>
+                    <my-global-icon iconName="support" aria-hidden="true"></my-global-icon>
                     <span class="icon-text" i18n>SUPPORT</span>
                   </div>
 
-                  <div (click)="showShareModal()" class="action-button" role="button">
+                  <div (click)="showShareModal()" (keyup.enter)="showShareModal()" class="action-button" tabindex="0" role="button" [ngbTooltip]="tooltipShare" placement="bottom">
```

Comment:
The tooltip is superfluous IMHO, since it is identical to the text, and the text is already pretty descriptive by itself.

Reply:
Oh sorry since there is a tooltip on the next save button I though it was missing one for the share button